.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
Was this intentional?

Reply:
Yes, otherwise VS Code uses a different TypeScript version.

> [ensure VS Code uses the same version of TypeScript as the project](https://github.com/vanilla-extract-css/vanilla-extract/pull/854/commits/b77eb7cab97c8aef9624030782e989a595e0e113)

<img width="370" alt="image" src="https://user-images.githubusercontent.com/3297808/193187558-c2d6db52-7a87-4b59-8024-b1f8eff71a75.png">

Reply:
👍